### PR TITLE
Copyedit

### PR DESCRIPTION
**Pull request by davorg**
I'm not sure you want to leave in the bit where Mohammad seems to be encouraging people to track down pirate copies of *Embedding and Extending Perl*.

### CODE REVIEW

File: src/545.json [hunk 1]
```diff
@@ -344,7 +344,7 @@
...
       "As for me, you might recall I've been learning Spanish for more than 3 years and started to learn Ladino (aka. Judeo-Spanish) a few weeks ago. Learning a foreign language is a totally different experience than learning a programming language. If for nothing else, because you don't get immediate feedback to your mistakes from a compiler. Well. Except if you use an application that helps you. My open source mindset quickly kicked in. I found an Open Source platform called <a href=\"https://librelingo.app/\">LibreLingo</a> that will help you learn languages. It is still in very early version, but started to use that platform to be able to practice Ladino and also started to contribute to the project. I'll share more about it when enough progress was made to show something in public. Until then you are invited to check it out.",
```
Comment (szabgab) [inline src/545.json:348]: Was this change ("ego" => "ago") intentional or some left over of last week where I fixed this too late. I think it should be "ego" not?

**Comment by szabgab** [comment]
yeah, I removed that suggestion to look for free copies. @manwar

File: src/545.json (hunk 1 above)
Comment (davorg) [reply src/545.json:348]: No, "ego" is correct. I missed it when dealing with the merge conflict. Sorry.

Comment (szabgab) [reply src/545.json:348]: :+1: np

**Pull request merged**